Gordon360/ApiControllers/ProfilesController.cs

```diff
@@ -361,33 +361,52 @@ public IHttpActionResult GetCliftonStrengths(string username)
         public IHttpActionResult GetMyImg()
         {
             var authenticatedUser = this.ActionContext.RequestContext.Principal as ClaimsPrincipal;
+            var username = authenticatedUser.Claims.FirstOrDefault(x => x.Type == "user_name").Value;
             var id = authenticatedUser.Claims.FirstOrDefault(x => x.Type == "id").Value;
-            var viewerName = authenticatedUser.Claims.FirstOrDefault(x => x.Type == "user_name").Value;
-            var viewerType = _roleCheckingService.getCollegeRole(viewerName);
-            var photoInfo = _profileService.GetPhotoPath(id);
+
+            var photoModel = _profileService.GetPhotoPath(id);
+
             string pref_img = "";
             string default_img = "";
+           
             var fileName = "";
             var filePath = System.Web.Configuration.WebConfigurationManager.AppSettings["DEFAULT_PREF_IMAGE_PATH"];
+
+            var unapprovedFileName = username + "_" + _accountService.GetAccountByUsername(username).account_id + ".jpg";
+            var unapprovedFilePath = System.Web.Configuration.WebConfigurationManager.AppSettings["DEFAULT_ID_SUBMISSION_PATH"];
+
             byte[] imageBytes;
             JObject result = new JObject();
 
-            //return default image if no photo info found for this user.
-            if (photoInfo == null)
+            if (photoModel != null)
+            {
+                // If this is the case, we just set the filename and complete the process after this if-else block
+                fileName = photoModel.Pref_Img_Name;
+            }
+            else if (File.Exists(unapprovedFilePath + unapprovedFileName))
+            {
+
+                var webClient = new WebClient();
+                imageBytes = webClient.DownloadData(unapprovedFilePath + unapprovedFileName);
+
+                string unapproved_img = Convert.ToBase64String(imageBytes);
+                result.Add("unap", unapproved_img);
```

Comment:
You're returning the image with the key "unap", but that isn't picked up anywhere in the front end. Every call to this route on the front end extracts only the "def" and "pref" images. So even if this were merged in, it wouldn't have any effect on the front end.